src/Samples/AuthoringDemo/AuthoringDemo/AuthoringDemo.csproj

```diff
@@ -25,10 +23,6 @@
         <PackageOutputPath>.\nuget</PackageOutputPath>
     </PropertyGroup>
     
-    <ItemGroup>
-        <Content Include="AuthoringDemo.targets" PackagePath="build;buildTransitive" />
-    </ItemGroup>
-    
     <ItemGroup>
       <Folder Include="nuget\" />
```

Comment:
Is this for the app?

Reply:
Yeah I'm generating the nupkg in this folder to make it easier for configuring package sources for the app